apps/moose-console/src/components/overview-list.tsx

```diff
@@ -20,9 +20,18 @@ export function PrimitivesOverviewList({ models }: { models: DataModel[] }) {
         <div className="col-span-3 xl:col-span-1 flex flex-col">
           <OverviewCard
             title="Flows"
-            numItems={0}
+            numItems={models.reduce(
+              (count, model) => count + (model.flows ? model.flows.length : 0),
+              0,
+            )}
             link="/primitives/flows"
-            items={[]}
+            items={models
+              .filter((model) => model.flows && model.flows.length > 0)
+              .slice(0, 4)
+              .map((model) => ({
+                name: model.model.name,
+                link: "",
```

Comment:
Maybe here we make the name a concat of Source -> Destination instead of just the model

Reply:
Ah, like this? What do you think @asl677 ?

<img width="2101" alt="Screenshot 2024-04-16 at 1 46 55 PM" src="https://github.com/514-labs/moose/assets/7286437/1c067656-3f1a-45c4-a9e5-904697a86e43">

Reply:
Spoke with @asl677 and we landed on using the format {source} to {destination} like this:

<img width="852" alt="Screenshot 2024-04-18 at 6 10 59 PM" src="https://github.com/514-labs/moose/assets/7286437/57d27f5a-2c29-4f8f-8747-7371d0d859ad">